client/src/pages/ClientDetailsPage.js

```diff
@@ -157,17 +164,32 @@ function ClientDetailsPage(props) {
               </InputGroup>
             </Col>
           </Row>
-          <Row className='my-3'>
+          <Row className="my-3">
             <CreateNewOrderButton clientID={clientID} />
           </Row>
+        </>
+      ) : (
+        <Row className="justify-content-around pt-2">
+          <Col className="ms-5">
+            <ClientDetails clientInfo={clientInfo} />
+          </Col>
+          <Col md="5" />
+        </Row>
+      )}
+
+      {isInitialized ? (
+        <>
           <Container>
             <Divider size={2} />
           </Container>
           <Row>
             <ClientOrders clientID={clientID} />
           </Row>
         </>
+      ) : (
+        ""

```

Comment:
It's better to use <></> or the form {condition && component}

`{isInitialized && <>WOW!</>}`